db/fixtures/server_roles.csv

```diff
@@ -2,6 +2,7 @@ name,description,max_concurrent,external_failover,role_scope
 automate,Automation Engine,0,false,region
 database_operations,Database Operations,0,false,region
 database_owner,Database Owner,1,false,database
+embedded_ansible,Embedded Ansible,1,false,zone
```

Comment:
I thought we were thinking of having an ansible zone in the region as the first step.  Limiting it to the region might box us into a corner when we want to deal with ansible clustering and multiple independent ansibles.  Maybe?